pulp_ansible/app/viewsets.py

```diff
@@ -378,7 +378,7 @@ class AnsibleRepositoryVersionViewSet(RepositoryVersionViewSet):
         responses={202: AsyncOperationResponseSerializer},
     )
     @action(detail=True, methods=["post"], serializer_class=AnsibleRepositoryRebuildSerializer)
-    def rebuild_metadata(self, request, pk):
+    def rebuild_metadata(self, request, *args, **kwargs):
```

Comment:
This change isn't doing a thing. Neither pk, nor args or kwargs are used anywhere.

Reply:
it's not used, but afaiu the method expects params `repository_pk` and `number`

https://github.com/pulp/pulpcore/blob/4e25949176d72c5dbe1c7623a9c47d253a18b085/pulpcore/app/viewsets/repository.py#L159 (for example: [pulpcore/app/viewsets/repository.py#L195](https://github.com/pulp/pulpcore/blob/4e25949176d72c5dbe1c7623a9c47d253a18b085/pulpcore/app/viewsets/repository.py#L195))

and since it's not used I decided to use the kwargs